cmd/bosun/conf/conf.go

```diff
@@ -27,47 +27,47 @@ import (
-	HTTPListen       string        // Web server listen address: :80
-	Hostname         string
-	RelayListen      string // OpenTSDB relay listen address: :4242
-	SMTPHost         string // SMTP address: ny-mail:25
-	SMTPUsername     string // SMTP username
-	SMTPPassword     string // SMTP password
-	Ping             bool
-	PingDuration     time.Duration // Duration from now to stop pinging hosts based on time since the host tag was touched
-	EmailFrom        string
-	StateFile        string
-	TimeAndDate      []int // timeanddate.com cities list
-	ResponseLimit    int64
-	SearchSince      opentsdb.Duration
-	UnknownTemplate  *Template
-	UnknownThreshold int
-	Templates        map[string]*Template
-	Alerts           map[string]*Alert
-	OrderedAlerts    []*Alert                 `json:"-"` //alerts in order they appear.
-	Notifications    map[string]*Notification `json:"-"`
-	RawText          string
-	Macros           map[string]*Macro
-	Lookups          map[string]*Lookup
-	Squelch          Squelches `json:"-"`
-	Quiet            bool
-	NoSleep          bool
-	AllowedPutIPs    []*net.IPNet
-	BlockedPutIPs    []*net.IPNet
-	ShortURLKey      string
+	Name               string        // Config file name
+	CheckFrequency     time.Duration // Time between alert checks: 5m
+	HTTPListen         string        // Web server listen address: :80
+	Hostname           string
+	RelayListen        string // OpenTSDB relay listen address: :4242
+	SMTPHost           string // SMTP address: ny-mail:25
+	SMTPUsername       string // SMTP username
+	SMTPPassword       string // SMTP password
+	Ping               bool
+	PingDuration       time.Duration // Duration from now to stop pinging hosts based on time since the host tag was touched
+	EmailFrom          string
+	StateFile          string
+	TimeAndDate        []int // timeanddate.com cities list
+	ResponseLimit      int64
+	SearchSince        opentsdb.Duration
+	UnknownTemplate    *Template
+	UnknownThreshold   int
+	Templates          map[string]*Template
+	Alerts             map[string]*Alert
+	OrderedAlerts      []*Alert                 `json:"-"` //alerts in order they appear.
+	Notifications      map[string]*Notification `json:"-"`
+	ActionNotification *Notification
+	RawText            string
+	Macros             map[string]*Macro
+	Lookups            map[string]*Lookup
+	Squelch            Squelches `json:"-"`
+	Quiet              bool
+	NoSleep            bool
+	AllowedPutIPs      []*net.IPNet
+	BlockedPutIPs      []*net.IPNet
+	ShortURLKey        string
 
 	TSDBHost             string                    // OpenTSDB relay and query destination: ny-devtsdb04:4242
 	GraphiteHost         string                    // Graphite query host: foo.bar.baz
 	LogstashElasticHosts expr.LogstashElasticHosts // CSV Elastic Hosts (All part of the same cluster) that stores logstash documents, i.e http://ny-elastic01:9200
 
-	tree            *parse.Tree
-	node            parse.Node
-	unknownTemplate string
```

Comment:
The removal of this has nothing to do with this feature. Can it go in a separate PR?